python/py-fipy/Portfile

```diff
@@ -34,9 +34,12 @@ if {${name} ne ${subport}} {
                         port:gmsh \
                         port:py${python.version}-future
 
-    post-destroot {
-        # Fix permissions problems on these files
-        xinstall -m 0644 -W ${worksrcpath}/FiPy.egg-info PKG-INFO SOURCES.txt dependency_links.txt entry_points.txt top_level.txt \
-            ${destroot}${python.pkgd}/FiPy-${version}-py${python.branch}.egg-info
-    }
+    ## this xinstall command is now causing an error -- I did not implement it,
+    ## and so I am not sure what is wrong with it, nor whether it is even still
+    ## needed, JJS 6/28/24
```

Comment:
`.egg-info` is no longer used in modern python packaging, so you can just delete this. The replacement is the `.dist-info` directory, so if you want to check that the issue is gone, you can check the permissions on the files installed in there.

Reply:
Looks OK to me:
```
$ ll /opt/local/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/FiPy-3.4.5.dist-info/
total 24
-rw-r--r-- 1 root wheel  1878 Jun 28 16:00 LICENSE.rst
-rw-r--r-- 1 root wheel  1838 Jun 28 16:00 METADATA
-rw-r--r-- 1 root wheel 26527 Jun 28 16:00 RECORD
-rw-r--r-- 1 root wheel    91 Jun 28 16:00 WHEEL
-rw-r--r-- 1 root wheel   205 Jun 28 16:00 entry_points.txt
-rw-r--r-- 1 root wheel    12 Jun 28 16:00 top_level.txt
```

Reply:
Ok, permissions look good to me:
```
$ ll /opt/local/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/FiPy-3.4.5.dist-info/
total 24
-rw-r--r-- 1 root wheel  1878 Jun 28 16:00 LICENSE.rst
-rw-r--r-- 1 root wheel  1838 Jun 28 16:00 METADATA
-rw-r--r-- 1 root wheel 26527 Jun 28 16:00 RECORD
-rw-r--r-- 1 root wheel    91 Jun 28 16:00 WHEEL
-rw-r--r-- 1 root wheel   205 Jun 28 16:00 entry_points.txt
-rw-r--r-- 1 root wheel    12 Jun 28 16:00 top_level.txt
```